### PR TITLE
Delta minutes

### PR DESCRIPTION
**Pull request by buzakovica**

### CODE REVIEW

File: common/to.etc.alg/src/main/java/to/etc/util/DateUtil.java [hunk 1]
```diff
@@ -590,6 +590,24 @@ public static int deltaInDays(Date start, Date end) {
...
 		return (int) ((double) (endL - startL) / (double) (1000 * 60 * 60 * 24));
 	}
+
+	/**
+	 * Get the difference in number of minutes between the specified dates.
+	 * @param start
+	 * @param end
+	 * @return
+	 */
+	public static int deltaInMinutes(Date start, Date end) {
```
Comment (millevlada) [inline common/to.etc.alg/src/main/java/to/etc/util/DateUtil.java:600]: hm, why we needed this? Should be simple as:
TimeUnit.MILLISECONDS.toMinutes(start.getTime() - end.getTime());

Wherever that was needed, just add it inline as this. No need for method here. What you saw here was some older code, it is not needed now with these newer java abilities.

**Review by millevlada — CHANGES_REQUESTED** [review]
this change is not needed, see remark

File: common/to.etc.alg/src/main/java/to/etc/util/DateUtil.java (hunk 1 above)
Comment (buzakovica) [reply common/to.etc.alg/src/main/java/to/etc/util/DateUtil.java:600]: You're right. Then I suggest deleting the PR?

**Comment by millevlada** [comment]
decided to remove this branch

**Pull request closed without merging**